test/clustertest/testplugin/TestPlugin.cpp

```diff
@@ -17,157 +22,192 @@ BedrockPlugin_TestPlugin::~BedrockPlugin_TestPlugin()
     delete httpsManager;
 }
 
+unique_ptr<BedrockCommand> BedrockPlugin_TestPlugin::getCommand(SQLiteCommand&& baseCommand) {
+    static set<string> supportedCommands = {
+        "testcommand",
+        "broadcastwithtimeouts",
+        "storeboradcasttimeouts",
+        "getbroadcasttimeouts",
+        "sendrequest",
+        "slowquery",
+        "httpstimeout",
+        "exceptioninpeek",
+        "generatesegfaultpeek",
+        "generateassertpeek",
+        "preventattach",
+        "chainedrequest",
+        "ineffectiveUpdate",
+        "exceptioninprocess",
+        "generatesegfaultprocess",
+        "idcollision",
+        "slowprocessquery",
+    };
+    for (auto& cmdName : supportedCommands) {
+        if (SStartsWith(baseCommand.request.methodLine, cmdName)) {
```

Comment:
why is this `SStartsWith` instead of `SIEqual`?

Reply:
Because https://github.com/Expensify/Bedrock/blob/39e88d253c2d5ab87a299d832f24ff63e97c655f/test/clustertest/tests/GracefulFailoverTest.cpp#L66